resources/js/Components/Articles/ArticleCard/ArticleCard.tsx

```diff
@@ -1,34 +1,30 @@
+import { DateTime } from "@ardenthq/sdk-intl";
 import { useTranslation } from "react-i18next";
-import { FeaturedCollections } from "@/Components/Articles/ArticleCard/ArticleCard.blocks";
-import { type ArticleCollections } from "@/Components/Articles/ArticleCard/ArticleCardContracts";
+import { FeaturedCollections } from "@/Components/Articles/Article.blocks";
+import { Img } from "@/Components/Image";
 
-interface Properties {
-    // TODO(@alfonsobries)[2023-09-30]. Replace with a dynamic type once defined
-    article: {
-        title: string;
-    };
-    collections: ArticleCollections;
-}
-
-export const ArticleCard = ({ article, collections }: Properties): JSX.Element => {
+export const ArticleCard = ({ article }: { article: App.Data.Articles.ArticleData }): JSX.Element => {
     const { t } = useTranslation();
 
     return (
         <a
             data-testid="ArticleCard"
-            href="#"
-            className="group flex w-full max-w-[335px] flex-col overflow-hidden rounded-xl border border-theme-secondary-300 bg-white"
+            href={`/articles/${article.slug}`}
```

Comment:
add a todo to change this to the route once those are available

Reply:
I'll handle it here: https://app.clickup.com/t/8678v0jgg. It is a small change, shouldn't cause too much headache.